002-quarkus-all-extensions/pom.xml

```diff
@@ -336,15 +297,10 @@
             <groupId>io.quarkus</groupId>
             <artifactId>quarkus-spring-data-jpa</artifactId>
         </dependency>
-        <!--        Caused by: java.lang.RuntimeException: java.lang.ClassCastException: io.sentry.servlet.SentryServletContainerInitializer cannot be cast to javax.servlet.ServletContainerInitializer-->
-        <!--      <dependency>-->
-        <!--        <groupId>io.quarkus</groupId>-->
-        <!--        <artifactId>quarkus-logging-sentry</artifactId>-->
-        <!--      </dependency>-->
-        <dependency>
-            <groupId>io.quarkus</groupId>
-            <artifactId>quarkus-azure-functions-http</artifactId>
-        </dependency>
+        <!--        <dependency>-->
+        <!--            <groupId>io.quarkus</groupId>-->
+        <!--            <artifactId>quarkus-azure-functions-http</artifactId>-->
+        <!--        </dependency>-->
```

Comment:
This one I don't understand. You are disabling it? If so, why? It's hard to track what is happening here.

Reply:
There is a conflict with other extensions, I will add comment